packages/daimo-api/src/contract/foreignCoinIndexer.ts

```diff
@@ -179,6 +183,7 @@ export class ForeignCoinIndexer extends Indexer {
         log.to,
         log.value.toString() as `${bigint}`,
         log.foreignToken,
+        daimoUSDC, // TODO: change to home coin
```

Comment:
👍 
we may be able to add homeCoin to `EAccount`, then this would be `toAcc.homeCoin`